providers/amazon/tests/system/amazon/aws/example_mwaa.py

```diff
@@ -77,12 +85,32 @@
     )
     # [END howto_sensor_mwaa_dag_run]
 
+    @task
```

Comment:
I am not against but this feels weird to me. System tests are DAGs and having a task that looks like a unit test feels really odd. I know you had a conversation in the other PR and again, I am not against. I just wanted to call it out. At least, if we go with that, I'd add comments to explain

Reply:
Added the comment!